cli/service.go

```diff
@@ -28,8 +29,6 @@ var (
 
 type ServiceWebCommand struct {
 	ConfigPath string `long:"conf" default:"/etc/mci_settings.yml" description:"path to the service configuration file"`
-	APIService bool   `long:"api" description:"run the API service (default port: 8080)"`
-	UIService  bool   `long:"ui" description:"run the UI service (default port: 9090)"`
```

Comment:
I think we need to get the service files edited to remove use of these flags before we remove them, or else the services will fail to start.